cardinal/option.go

```diff
@@ -63,3 +66,8 @@ func WithLoopInterval(interval time.Duration) WorldOption {
 		},
 	}
 }
+
+func WithPrettyLogOption(world *ecs.World) {
```

Comment:
This should be `func WithPrettyLog(world *ecs.World) WorldOption`. It's functionally the same thing, but by returning a WorldOption this function will get grouped with the other options in generated godocs.